views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -34,6 +34,60 @@ define([
 ], function($, __, pluginFactory, hider, previewerFactory, buttonTpl){
     'use strict';
 
+    let saved = true;
```

Comment:
This should be inside `pluginFactory` because now it will be shared between all created preview plugins

Reply:
done